homeassistant/components/twitch/sensor.py

```diff
@@ -151,8 +149,15 @@ def update(self) -> None:
             self._attr_extra_state_attributes[ATTR_GAME] = stream["game_name"]
             self._attr_extra_state_attributes[ATTR_TITLE] = stream["title"]
             self._attr_entity_picture = stream["thumbnail_url"]
+            if self._attr_entity_picture is not None:
+                self._attr_entity_picture = self._attr_entity_picture.replace(
+                    "{width}", "24"
+                ).replace("{height}", "24")
```

Comment:
We can use formatting in this case (instead of replacing).
```suggestion
                self._attr_entity_picture = self._attr_entity_picture.self._attr_entity_picture = thumbnail_url.format(
                    height=24,
                    width=24,
                  )
```

Reply:
Oh, you hipsters with your new Python stuff. :-)

Reply:
Also seems your suggested code change had "minor issues". :-)